### PR TITLE
Change the way model file (and label file) are loaded for object detector and face detector 

### PR DESCRIPTION
**Pull request by shmpwk**
First, I changed the way model file and label file are specified for object detector.

Originally, they are parameters launching from the beginning. But I wonder they can be changed during launching. 
For example, originally, the robot loads them at startup. When we want to use our custom model file, we have to restart robot and it is troublesome.
 
With the help of this PR, we can dynamically change the model file and label file even while the robot launching.

I also change the way model file are specified for face detector. 
I need to change `detector_base.py` for object detector. 
Since both `object_detector.py` and `face_detector.py` are based on `detector_base.py`, the way model file (and label file) are loaded should be the same for both.

### CODE REVIEW

**Comment by shmpwk** [comment]
I tested the object detector and face detector with my xtion camera, but I haven't tried with 
- PR2 kinnect camera
- panorama camera

So I have to make sure they work properly before merging PR.

File: python/coral_usb/detector_base.py [hunk 1]
```diff
@@ -65,13 +65,7 @@ def __init__(self, model_file=None, label_file=None, namespace='~'):
                 rospy.logwarn(
                     'device {} is already assigned: {}'.format(
                         device_id, device_path))
-
-        self.engine = DetectionEngine(self.model_file, device_path=device_path)
-        if label_file is None:
-            self.label_ids = None
-            self.label_names = None
-        else:
-            self.label_ids, self.label_names = self._load_labels(label_file)
```
Comment (knorth55) [inline python/coral_usb/detector_base.py:74]: please do not remove this line

File: python/coral_usb/detector_base.py [hunk 2]
```diff
@@ -129,6 +123,13 @@ def visualize(self):
     def config_callback(self, config, level):
         self.score_thresh = config.score_thresh
         self.top_k = config.top_k
+        self.model_file = config.model_file
+        try:
```
Comment (knorth55) [inline python/coral_usb/detector_base.py:127]: please do not use try.

**Review by knorth55 — CHANGES_REQUESTED** [review]

File: python/coral_usb/object_detector.py [hunk 3]
```diff
@@ -10,14 +9,7 @@
...
     def __init__(self, namespace='~'):
-        rospack = rospkg.RosPack()
-        pkg_path = rospack.get_path('coral_usb')
-        model_file = os.path.join(
-            pkg_path,
-            './models/mobilenet_ssd_v2_coco_quant_postprocess_edgetpu.tflite')
-        label_file = os.path.join(pkg_path, './models/coco_labels.txt')
-        super(EdgeTPUObjectDetector, self).__init__(
-            model_file, label_file, namespace)
+        super(EdgeTPUObjectDetector, self).__init__(namespace)
```
Comment (knorth55) [inline python/coral_usb/object_detector.py:12]: it seems totally wrong. did you really try it?

**Review by knorth55 — CHANGES_REQUESTED** [review]

File: python/coral_usb/detector_base.py (hunk 1 above)
Comment (shmpwk) [reply python/coral_usb/detector_base.py:74]: I removed it for face detector which doesn't have label file but specifies label_ids and label_names. See [here](https://github.com/knorth55/coral_usb_ros/blob/11a9977552f5769f5c82c77e5e9b71207f5c51b5/python/coral_usb/face_detector.py#L15-L16).

File: python/coral_usb/detector_base.py (hunk 2 above)
Comment (shmpwk) [reply python/coral_usb/detector_base.py:127]: You mean I should use if-else?

File: python/coral_usb/object_detector.py (hunk 3 above)
Comment (shmpwk) [reply python/coral_usb/object_detector.py:12]: Yes, with my xtion camera.

Comment (shmpwk) [reply python/coral_usb/object_detector.py:12]: I think I should also change from 
`def __init__(self, model_file=None, label_file=None, namespace='~'):`
to
`def __init__(self, namespace='~'):`
 [here](https://github.com/knorth55/coral_usb_ros/blob/11a9977552f5769f5c82c77e5e9b71207f5c51b5/python/coral_usb/detector_base.py#L38)

**Comment by knorth55** [comment]
I update this PR.

File: python/coral_usb/detector_base.py (hunk 2 above)
Comment (knorth55) [reply python/coral_usb/detector_base.py]: yes

Comment (shmpwk) [reply python/coral_usb/detector_base.py]: yes from knorth55

File: python/coral_usb/object_detector.py (hunk 3 above)
Comment (shmpwk) [reply python/coral_usb/object_detector.py]: > I think I should also change from
> `def __init__(self, model_file=None, label_file=None, namespace='~'):`
> to
> `def __init__(self, namespace='~'):`https://github.com/knorth55/coral_usb_ros/pull/58/commits/513c4499c9cdb92c1b2f7da40cc08b41e0bb1328
> [here](https://github.com/knorth55/coral_usb_ros/blob/11a9977552f5769f5c82c77e5e9b71207f5c51b5/python/coral_usb/detector_base.py#L38)

is not wrong but no good.

**Comment by shmpwk** [comment]
Thank you. 
I checked object and face detector with my xction camera.
With or without model and label file arguments when launching, it properly works.

**Comment by shmpwk** [comment]
Thank you, I confirmed to work it properly.

**Comment by shmpwk** [comment]
Both human pose estimator and semantic segmenter work properly.

**Pull request merged**